README.md

```diff
@@ -7,3 +7,19 @@ chemtrail-distribution system based on flight paths supplied by the user and the
 to carry that payload alongside regular passengers. Results should be in the form of square meters covered and
 energy cost in Joules, as well as an optional visual representation on a map and the density of the gas at ground level. 
 The threshold gas level for visualization on the ground, as well as the plane height, should be specifiable by the user.
+
+# Installation Instruction
+
+# User Guide
+
+# What did we learn
+
+Our main idea when joining the Hackathon was to create something interesting and having fun.
+Here some bullet Points of our main lessons we learned: 
+
+• Collaborate on GitHub effectively 
```

Comment:
```suggestion
• Collaborate on GitHub effectively 
* Practice working with scrum and sprints during the development 
```

Reply:
mb practice working in agile environment ?

Reply:
I believe they will love to see those words :p